opentak/logger.py

```diff
@@ -0,0 +1,61 @@
+# mypy: disable-error-code="attr-defined, method-assign"
```

Comment:
IMHO we should delete the logger specification. We can only leave the `logger = logging.getLogger("tak")` and let anyone who use thise package to specify a logging configuration